### PR TITLE
Feature/low pop tutorial

### PR DESCRIPTION
**Pull request by UlricVilleneuve**

### CODE REVIEW

File: Eskillate/Assets/Resources/Core/Labels/es/DragAndDropDescription.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"Label": "Tiene que arrastrar las formas y colocar las sobres los espacios correspondientes."
```
Comment (princess-chichie) [inline Eskillate/Assets/Resources/Core/Labels/es/DragAndDropDescription.json:2]: **colocarlas** no colocar las

File: Eskillate/Assets/Resources/Core/Labels/es/HideAndSeekDescription.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+	"Label": "Tiene que encontrar el objeto pedido entre el monton."
```
Comment (princess-chichie) [inline Eskillate/Assets/Resources/Core/Labels/es/HideAndSeekDescription.json:2]: Perdido

File: Eskillate/Assets/Resources/Core/Labels/es/LowPopDescription.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+	"Label": "Tiene que hacer estallar las burbujas de la mas pequeña a la mas grande."
```
Comment (princess-chichie) [inline Eskillate/Assets/Resources/Core/Labels/es/LowPopDescription.json:2]: más

File: Eskillate/Assets/Scripts/Core/AudioScript.cs [hunk 4]
```diff
@@ -1,39 +1,42 @@
 ﻿using UnityEngine;
 
-public class AudioScript : MonoBehaviour
+namespace Core
```
Comment (UlricVilleneuve) [inline Eskillate/Assets/Scripts/Core/AudioScript.cs:3]: I think some of these changes come from [this PR](https://github.com/ulria/Eskillate/pull/12#pullrequestreview-662690956). Be sure to update your branch once it is merged.

Comment (UlricVilleneuve) [reply Eskillate/Assets/Scripts/Core/AudioScript.cs:3]: Review of the files below the labels that are not  marked as view still needs to be done

**Pull request merged**